mlflow/java/src/main/java/org/mlflow/Flavor.java

```diff
@@ -0,0 +1,13 @@
+package org.mlflow;
```

Comment:
Can we remove the file `.swn` files? What are those? Should we add them to a gitignore?